tests/util.py

```diff
@@ -73,6 +94,18 @@ def from_url(url: str, name: str | None = None):
         name = name or get_url_file_name(url)
         return ModelFile(name).download(url)
 
+    @staticmethod
+    def from_url_zip(
+        url: str, name: str | None = None, rel_model_dir: Path | None = None
+    ):
+        path = download_model(url, "temp.zip")
+        print(f"Extracting {path}...")
+        extract_zip(
+            path, rel_model_dir or Path(name or ""), name or get_url_file_name(url)
+        )
+        os.remove(path)
+        return ModelFile(name or get_url_file_name(url))
```

Comment:
1. `rel_model_dir` is not a directory.
2. `rel_model_dir` should also accept string parameters.
3. Could we make `rel_model_dir` a required parameter and derive `name` from it (if not given)? This means that you could then write `from_url_zip("url...", "path/to/model.pth")` and it will automatically use the correct name.
4. This downloads the zip every time you run a test. You need to check for an existing model file at the start of the function.